jsHelper/spicetifyWrapper.js

```diff
@@ -439,6 +447,25 @@ window.Spicetify = {
 
 	if (Spicetify.Color) Spicetify.Color.CSSFormat = modules.find(m => m?.RGBA);
 
+	// Combine snackbar and notification
+	(async function bindShowNotification() {
+		if (!(Spicetify.Snackbar && Spicetify.notification)) {
+			setTimeout(bindShowNotification, 10);
+			return;
+		}
+
+		Spicetify.showNotification = (message, isError = false, msTimeout, snack = true) => {
+			if (snack) {
+				Spicetify.Snackbar.enqueueSnackbar(message, {
+					variant: isError ? "error" : "default",
+					autoHideDuration: msTimeout
+				});
+			} else {
+				Spicetify.notification(message, isError, msTimeout);
+			}
+		};
```

Comment:
That's not what I meant
```suggestion
		Spicetify.showNotification = (message, isError = false, msTimeout) => {
			if (!Spicetify.Snackbar?.enqueueSnackbar) {
				Spicetify.notification(message, isError, msTimeout);
				return;
			}
			
			Spicetify.Snackbar.enqueueSnackbar(message, {
				variant: isError ? "error" : "default",
				autoHideDuration: msTimeout
			});
		};
```

Reply:
isn't that the same thing just with the inability for devs to override it? guess that works regardless though 👍

Reply:
true, sounds good to me then